frame/contracts/src/wasm/runtime.rs

```diff
@@ -623,6 +623,17 @@ define_env!(Env, <E: Ext>,
 		Ok(())
 	},
 
+	// Stores the balance required to create a tombstone upon contract eviction
+	// into the scratch buffer.
```

Comment:
We might want to expand that this deposit is on top of the existential deposit.
In other words, in order for a contract to leave a tombstone the balance of the contract should not go below the sum of existential deposit and the tombstone deposit. This sum is commonly referred as subsistence threshold in code.

Reply:
done